### PR TITLE
Add a key feature to README

### PR DESCRIPTION
**Pull request by Grace-codes**
## Summary

This PR adds a key feature to README

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by Auxylotl** [comment]
buttons + dropdown too?

**Comment by Grace-codes** [comment]
I'll add that too

**Review by CodeWithSwastik — APPROVED** [review]

**Comment by nerdguyahmad** [comment]
Are these *really* needed? :thinking: 

I mean, A key feature like "Provides entire API coverage" or "Implements the entire Discord API" would be better.

**Comment by Grace-codes** [comment]
It is. People keep on asking if pycord has slash commands or buttons. And there's people like this 
image

**Comment by nerdguyahmad** [comment]
> It is. People keep on asking if pycord has slash commands or buttons. And there's people like this
> image

Scrolling down by few inches reveals slash commands examples. :smile:

File: README.rst [hunk 1]
```diff
@@ -19,6 +19,7 @@ Key Features
 - Modern Pythonic API using ``async`` and ``await``.
 - Proper rate limit handling.
 - Optimised in both speed and memory.
+- Supports Slash Commands, Context Menus and Message Components.
```
Comment (JacobMonck) [inline README.rst:22]: I think we should have something like "Supports Slash Commands, Context Menus and Message Components via gateway". I am not sure though if we do slash commands through http or the gateway.

Comment (Dorukyum) [reply README.rst:22]: All events are received through the gateway, altough commands are registered through http requests.

**Review by Middledot — APPROVED** [review]

**Review by BobDotCom — APPROVED** [review]

**Pull request merged**